src/extension/messages.ts

```diff
@@ -90,7 +90,7 @@ export type ClientMessage =
       type: "connectToDevtools";
       payload: ClientContext;
     }
-  | { type: "disconnectFromDevtools" }
+  | { type: "clientTerminated" }
```

Comment:
With the change in this PR, it felt appropriate to rename this message since a terminated client is the only thing that sent this message. This now better represents the actual thing that happened.